src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/mainscreen/viewmodel/MainViewViewModel.kt

```diff
@@ -49,29 +58,34 @@ class MainViewViewModel : ViewModel() {
         }
     }
 
-    fun projectSelected(selectedProject: Collection) {
+    private fun projectSelected(selectedProject: Collection) {
         setActiveProjectText(selectedProject)
 
         find<MainScreenView>().activeFragment.dock<CollectionsGrid>()
         CollectionsGrid().apply {
             activeProject.bindBidirectional(selectedProjectProperty)
             activeCollection.bindBidirectional(selectedCollectionProperty)
+            activeResource.bindBidirectional(selectedResourceProperty)
         }
     }
 
-    fun collectionSelected(collection: Collection) {
+    private fun collectionSelected(collection: Collection) {
         setActiveCollectionText(collection)
 
         find<MainScreenView>().activeFragment.dock<ContentGrid>()
         ContentGrid().apply {
             activeProject.bindBidirectional(selectedProjectProperty)
             activeCollection.bindBidirectional(selectedCollectionProperty)
+            activeResource.bindBidirectional(selectedResourceProperty)
             activeContent.bindBidirectional(selectedContentProperty)
-
         }
     }
 
-    fun contentSelected(content: Content) {
+    private fun resourceSelected(resource: ResourceMetadata) {
+        // TODO: refilter collections and change colors in CollectionGrid here
```

Comment:
Why wouldn't we just do this in the CollectionsGridViewModel?

Reply:
We probably would (and I think I already do). Sorry for dumping half-baked code on you.

Reply:
@aunger I think I'll go ahead and merge the PR into my branch- perhaps this todo message can just be changed to something like "Consider setting a shared `cssRule` property here"